base/logger/DESCRIPTION

```diff
@@ -11,5 +11,5 @@ Suggests: testthat
 License: BSD_3_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.0
```

Comment:
Please keep Roxygen at version 7.0.2 for now -- I know it's annoying to keep the version behind on a dev machine, but when we bump the version we want to do it in all packages at once.